setup.py

```diff
@@ -57,6 +75,10 @@ def build_extension(self, ext):
 cmdclass = versioneer.get_cmdclass()
 cmdclass.update(build_ext=CMakeBuild)
 
+external_modules = None
```

Comment:
nit: naming: I think `ext_modules` stands for "extension modules" (could just use the same name as the kwarg?)

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/944/commits/7d372f6d16783b07cb916a1f350e6efb4a3b32a3